cli/azd/internal/appdetect/spring_boot.go

```diff
@@ -98,6 +98,8 @@ func detectAzureDependenciesByAnalyzingSpringBootProject(
 	detectEventHubs(azdProject, &springBootProject)
 	detectStorageAccount(azdProject, &springBootProject)
 	detectSpringCloudAzure(azdProject, &springBootProject)
+	detectSpringCloudEureka(azdProject, &springBootProject)
```

Comment:
detect Eureka and Config related dependencies, and append them into `Project.Dependencies`